### PR TITLE
Fix repeat alert feature. Add "Enabled" flag to alert types. Round al…

### LINKED ISSUE #159: User gets low and high alerts within 1 minute while glucose level > 25000
Looks like Low and High alert received simultaneously when there are no data at all
Screenshot at Sep 30 18-14-23

Screenshot at Sep 30 17-42-52

### LINKED ISSUE #176: Round off numbers to one digit after the dot on the main screen graph when units are mmol/L
Screenshot at Oct 05 17-38-33

### LINKED ISSUE #189: Adjust basic alert algorithm because now user gets notifications to often
As were discussed let's use the Fibonacci sequence

### LINKED ISSUE #196: User gets two alerts simultaneously “level is low” and “level is urgent low” when level on the edge of two ranges
Please check if it works as expected. 
I got two messages a few times and after this recieved only “level is urgent low” (probably when glucose level went lower)
AC43A8C0-DFF4-41B8-9B9F-16B4864FC8D7
7C3C561A-FCD1-4F27-8CE0-F373FBF955E5
"Range Selection" values at the moment

Screenshot at Oct 14 15-43-44

### LINKED ISSUE #198: Would be nice to have an alert about isn't running app, for example was closed accidentally
Example (If I'm not mistaking the alert is shown only once after 10 min when app was close):
Screenshot at Oct 14 16-27-04

### LINKED ISSUE #199: Glucose values should be integers not fractional when mg/dL are selected in Settings
Screenshot at Oct 14 16-01-39

### LINKED ISSUE #200: Remove "Repeat" option from alerts where it's unnecessary and add possibility to disable an alert
Screenshot at Oct 14 18-44-08

### LINKED ISSUE #203: High glucose alerts are coming every minute even without "Repeat" option selected
High alert settings

Screenshot at Oct 15 19-38-01

### PR DESCRIPTION
**Pull request by Ivan-Skoryk**
…l displaying glucose values to 1 digit after decimal point.

## Summary
 
 
Closes #203 
Closes #200 
Closes #199 
Closes #196 
Closes #176 
Closes #159 
Closes #198 
Closes #189 

 #### Checklist:
 - [x] I have added a brief description of the problem in the PR body.
 - [x] I have explained how I solved the problem in the PR body.
 - [x] I have covered possible edge cases and side effects.
 - [x] I have have attached screenshots to the PR of any UI change.
 - [x] I have checked that my changes work on iOS and macOS targets.
 - [x] I have checked that my changes work correctly on devices with and without a Home button.
 - [x] I have added unit tests for the new logic.
 - [x] I have achieved maximum possible code coverage for the new code.
 - [x] I have checked that my UI works correctly on both Light and Dark themes.

 #### Acceptance Criteria
 

 #### Screenshots or GIFs (if appropriate)

### CODE REVIEW

File: xDrip/Controllers/Notifications/NotificationController.swift [hunk 1]
```diff
@@ -184,6 +234,68 @@ final class NotificationController: NSObject {
...
+                return false
+            }
+            alertSentCount[type]? += 1
+            alertSkipCount[type] = fibonicci(alertSentCount[type] ?? 0) - 1
+        }
+        
+        return true
+    }
+    
+    private func fibonicci(_ number: Int) -> Int {
```
Comment (Artem-Kalmykov) [inline xDrip/Controllers/Notifications/NotificationController.swift]: `fibonacci`

File: xDrip/Model/Glucose/Calibration.swift [hunk 2]
```diff
@@ -221,6 +221,7 @@ final class Calibration: Object {
         calculateWLS()
         adjustRecentReadings(30)
         NightscoutService.shared.scanForNotUploadedEntries()
+        NotificationCenter.default.post(name: NSNotification.Name("RegularCalibrationCreated"), object: nil)
```
Comment (Artem-Kalmykov) [inline xDrip/Model/Glucose/Calibration.swift]: Why `NS` and why not in a constant?

File: xDrip/Model/User/Settings/AlertConfiguration.swift [hunk 3]
```diff
@@ -10,7 +10,8 @@ import Foundation
 import RealmSwift
 
 final class AlertConfiguration: Object {
-    @objc private(set) dynamic var isEnabled: Bool = false
+    @objc private(set) dynamic var isOverride: Bool = false
```
Comment (Artem-Kalmykov) [inline xDrip/Model/User/Settings/AlertConfiguration.swift]: `isOverriden`

**Review by Artem-Kalmykov — CHANGES_REQUESTED** [review]

**Review by Artem-Kalmykov — APPROVED** [review]

File: xDrip/Model/User/Settings/AlertConfiguration.swift [hunk 4]
```diff
@@ -10,7 +10,8 @@ import Foundation
...
+    @objc private(set) dynamic var isEnabled: Bool = true
```
Comment (Ivan-Skoryk) [inline xDrip/Model/User/Settings/AlertConfiguration.swift:14]: @qwqatest alerts already enabled by default
#200

**Pull request merged**